validators/node-cli.md

```diff
@@ -0,0 +1,264 @@
+## SKALE Node CLI
+
+Note: This is **an insecure pre-release** software specifically for Alpine team members. 
+
+See the SKALE Node CLI code and documentation on [**GitHub**](https://github.com/skalenetwork/skale-node-cli)*‍  
+
+This document contains instructions on how to get started with the SKALE Node CLI.  
+
+### **Prerequisites**
+
+-   A Linux x86_64 machine
+-   SGX-enabled Intel processor
+-   Ports 22, 8080, 9100, and 10000-11000, and ICMP IPv4 open for all
+-   Ubuntu 16.04 or later LTS
+-   200GB attached storage (mainnet requirements will be defined soon)
+-   32GB RAM  
+
+This pre-release Validator and Node software is insecure. As such, the only tokens running on this early phase Validator net are  _test tokens only_. SKALE will release a more secure system prior to later Validator Devnet releases.  
+‍  
+If you have any concerns or questions, please do not hesitate to reach out to SKALE Team leads on [discord](http://skale.chat/).  
+
+[![Discord](https://img.shields.io/discord/534485763354787851.svg)](https://discord.gg/vvUtWJB)
+
+### Step 1: Install SKALE Node CLI
+
+#### Download the SKALE Node CLI binary
+
+Replace version number with `0.8.0-develop.20`
+
+**Terminal Command:**
+
+```bash
+VERSION_NUM=[Version Number] && sudo -E bash -c "curl -L https://skale-cli.sfo2.cdn.digitaloceanspaces.com/beta/skale-$VERSION_NUM-`uname -s`-`uname -m` >  /usr/local/bin/skale"
+
+```
+
+#### Make the SKALE Node CLI binary executable
+
+**Terminal Command:**
+
+```bash
+sudo chmod +x /usr/local/bin/skale
+
+```
+
+### Step 2: Setup SKALE Node
+
+#### Step 2.1: Initialize SKALE node daemon and install dependencies
+
+Required options for the `skale node init` command:
+
+-   `--disk-mountpoint` - Mount point of the disk to be used for storing sChains data
+-   `--sgx-url` - URL to SGX server in the network, can be used for current node if the current node supports SGX-enabled Intel processor
+-   `--env-file` - path to env file where required parameters listed above are defined
+
+Required options for the `skale node init` command in environment file:
+
+-   `SGX_SERVER_URL` - URL to SGX server in the network, can be used for current node if the current node supports SGX
+-   `DISK_MOUNTPOINT` - Mount point of the disk to be used for storing sChains data
+-   `IMA_CONTRACTS_INFO_URL` - URL to IMA contracts ABI and addresses
+-   `MANAGER_CONTRACTS_INFO_URL` - URL to SKALE Manager contracts ABI and addresses
+-   `FILEBEAT_HOST` - URL to the Filebeat log server
+-   `GITHUB_TOKEN` - token for accessing `skale-node` repo
+-   `GIT_BRANCH` - git branch used for initialization
+-   `DOCKER_PASSWORD` - password for DockerHub
```

Comment:
docker_password and docker_username is deprecated, see https://github.com/skalenetwork/skale-node-cli/pull/166